unfoldingWord/translationCore
# Added correct instructions for tools main page and login main page

#### This pull request addresses:
- Added correct instructions for tools main page. #976 
- Added correct instructions for user main page. #1985 
- Update grammar on the `log out button` #2173

#### How to test this pull request:
- Go to the tools Main page and verify the instructions read as **Select a tool from the list**.

- Login with your `door43` and `local user` act and see how the instructions update accordingly.
- Log out with from both `door43` and `local user`

---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2241)
